Competition2017/src/competition/subsystems/drive/commands/DriveForDistanceCommand.java

```diff
@@ -24,6 +24,8 @@
     private DoubleProperty deltaDistanceProp;
     private ContiguousHeading targetHeading;
     
+    public static String driveToPositionPidName = "Drive to position";
```

Comment:
shouldn't this then get referenced in the pidmanager construction? Otherwise they could get out of sync?
while we're here it's a little strange that this name doesn't match either the class or the variable name in code of the property, wonder if there's some cheap rationalize we could do there?